### ISSUE TITLE
Lint/LiteralInInterpolation incorrectly flags interpolation of backticks

### ISSUE THREAD

**Issue by aevernon** [bbatsov/rubocop#2647]
### Steps to Reproduce

```bash
$ echo 'puts "Backticks #{`date`}"' > foo.rb
$ rubocop foo.rb 
```

### Observed Behavior

```
foo.rb:1:19: W: Literal interpolation detected.
puts "Backticks #{`date`}"'
                  ^^^^^^
```

### Expected Behavior

Interpolation of dynamic string literals should be allowed.

### Environment

```bash
$ rubocop -V
0.36.0 (using Parser 2.3.0.1, running on ruby 2.2.4 x86_64-linux)
```

**Comment by segiddins** [comment]
I can't reproduce the issue with the given code sample
